src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindCollectionsCommandResolver.java

```diff
@@ -13,13 +13,11 @@
 /** Command resolver for the {@link FindCollectionsCommand}. */
 @ApplicationScoped
 public class FindCollectionsCommandResolver implements CommandResolver<FindCollectionsCommand> {
-
-  private final SchemaManager schemaManager;
   private final ObjectMapper objectMapper;
+  @Inject CQLSessionCache cqlSessionCache;
```

Comment:
Same as above, remove `@Inject`, make final. But also need to be added to injected via constructor.